docs/config.rst

```diff
@@ -48,6 +48,11 @@ core
 
     If ``false``, Gordon will exit out if it can't load one or more plugins.
 
+.. option:: route=DICT-OF-STRINGS
+    The route message should take through Gordon, defined via the phases each
+    plugin implements. Route should be a dict key-value pairs of
+    ``{'from-phase': 'to-phase'}``.
```

Comment:
This is helpful, but the example should probably be in `toml` format, like you have added in the example:

 ```toml
[core]
plugins = ["one.plugin", "two.plugin"]
debug = false
[core.route]
from_phase1 = "to_phase2"
from_phase2 = "to_phase3"
```

So then it should have a separate heading like `[core.logging]` below, and therefore the option documentation is just key="value" sort of thing.

Reply:
I started with that, but then wasn't sure what to do name the .. option part, since the keys can be whatever.  Should I put something like `.. option:: from_phase=STR`? Or is there another way to indicate the keys are arbitrary?

Reply:
I'm not aware of the best way to do that; I don't know of any project off the top of my head that allows arbitrary keys. But I think what you've proposed is what I would have done, noticing that multiple are allowed. I wonder if it's worth saying keys should be unique (for now)?